ckanext/spark/public/spark-frtend.css

```
@@ -1,15 +1,663 @@
+}
+
+.dataset-label {
+    background-color: #00A99E;
+    color: #FFFFFF;
+    text-transform: uppercase;
+    font-family: "Plus Jakarta Sans", sans-serif;
+    font-weight: normal;
+}
+
+.dataset-details {
+    background-color: #FFFFFF;!important;
```

Stylelint (17.14.0)

[error] Unknown word !important (CssSyntaxError)

(CssSyntaxError)